File: 10_Armstrong/armstrong.py
num = int(input("Enter your number: "))
temp1 = 0
temp2=num

while(temp2 != 0):
    temp1 = (temp2 % 10)**3 + temp1
    temp2 = temp2 // 10
    
if(temp1==num):
    print("Number is armstrong!")
else:
    print("Number is not armstrong!")

File: 10_Armstrong/armstrong.c
# include<stdio.h>

int main(int argc, char const *argv[])
{
    int num, temp1=0, temp2, temp3;

    printf("Enter your number: ");
    scanf("%d", &num);

    temp2 = num;

    while(temp2 != 0){
        temp1 = temp2%10;
        temp2 = temp2/10;
        temp3 = (temp1 * temp1 * temp1) + temp3;
    }

    if(temp3 == num){
        printf("Number is armstrong!");
    }else{
        printf("Number is not armstrong");
    }
    
    return 0;
}
